server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java

```diff
@@ -127,6 +128,15 @@ public class IndexMetadata implements Diffable<IndexMetadata>, ToXContentFragmen
         EnumSet.of(ClusterBlockLevel.WRITE)
     );
 
+    public boolean isRollupIndex() {
```

Comment:
I have the feeling this is not the best I can do here. I am working on a better way to do this at the moment but I wanted to push this change because it might be good enough. If that feeling is shared but this is considered "good enough" we could proceed merging this to unblock Kibana folks experimenting with catching exceptions. Then I can create another issue to refactor this. N.B. I am on vacation next week and Christos is on vacation too and I would like to avoid Kibana folks being stuck waiting for this.

Reply:
I think it's cool as a temporary thing. Maybe a `TODO` in the code so we know we thought it was temporary when we added it.